napari/_qt/dialogs/qt_notification.py

```diff
@@ -108,6 +109,28 @@ def __init__(
         self.geom_anim = QPropertyAnimation(self, b"geometry", self)
         self.move_to_bottom_right()
 
+    def _update_icon(self, severity: str):
+        """Update the icon to match the severity level."""
+        from ...settings import get_settings
+        from ...utils.theme import get_theme
+
+        settings = get_settings()
+        theme = settings.appearance.theme
+        default_color = getattr(get_theme(theme, False), 'icon')
+
+        # FIXME: Should these be defined at the theme level?
+        # Currently there is a warning one
+        colors = {
+            'error': "#D85E38",
+            'warning': "#E3B617",
+            'info': default_color,
+            'debug': default_color,
+            'none': default_color,
```

Comment:
Should we add/create other icons for info and debug @isabela-pf ?

Reply:
I know this has already been merged, but are you asking about creating icons for `info`, `debug`? I can if there is interest, I would just want to check that they are used the ways the name imply.

However I also know there's been talk about an icon overhaul in issues like #3136, so I'm not sure if it's more or less helpful to do this work when we aren't looking at it from the big picture. If other people agree now's a good time to add in these other icons, I'm happy to make it happen.

Reply:
@napari/core-devs  :) ?